app/jobs/resolution_proofing_job.rb

```diff
@@ -102,6 +115,9 @@ def proof_lexisnexis_ddp_with_threatmetrix_if_needed(
     ddp_pii = applicant_pii.dup
     ddp_pii[:threatmetrix_session_id] = threatmetrix_session_id
     ddp_pii[:email] = user&.confirmed_email_addresses&.first&.email
+    ddp_pii[:input_ip_address] = request_ip
+    ddp_pii[:local_attrib_1] = uuid_prefix
```

Comment:
these names don't match what the proofer is looking for:
```suggestion
    ddp_pii[:request_ip] = request_ip
    ddp_pii[:uuid_prefix] = uuid_prefix
```